docker_image/lib/config.guess

```diff
@@ -0,0 +1,1698 @@
+#! /bin/sh
```

Comment:
I don't think this file is relevant, can you explain the purpose of this huge amount of code?

Reply:
It resolves an error regarding compiling TA-lib on ARM. My raspberry pi 4 won't compile TA-lib without this file in all images i've tried with Ta-lib.

It's a newer file than what's in the .tar package which cannot figure out arm

Reply:
I understand but it's only for raspberry then nobody is concerned about this, especially because this program required huge CPU performance to run well. I prefer to keep it simple for the most use case, you can create a separate repo for raspberry, it may help the community.